configs/clustering/2021.04.01.ini

```diff
@@ -0,0 +1,26 @@
+[DEFAULT]
+version = 2021.04.01
```

Comment:
no - the numbers are a bit problematic. It is `01` since this is the first clustering config i had in april 2021. But I do agree it'd be better to have a uniform number used to tie the configs together